fusionfrontend/src/app/store/oisp/oisp-alert/oisp-alert.model.ts

```diff
@@ -16,7 +16,7 @@
 import { ID } from '@datorama/akita';
 
 export class OispAlert {
-  id: ID;
+  id: ID;  // actually not part of model, but necessary for storing with upsertMany using akita
```

Comment:
here you hat to change the @SotreConfig -> https://datorama.github.io/akita/docs/entities/entity-store/ to use custom ids